dockers/docker-orchagent/supervisor-proc-exit-listener

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import os
+import signal
+import sys
+import syslog
+
+from supervisor import childutils
+
+# Process names as defined in supervisor.conf file
+critical_processes = ['orchagent']
```

Comment:
should add other process such as vlanmgrd, portsyncd, and other critical process.

Reply:
Done in https://github.com/Azure/sonic-buildimage/pull/2546/commits/438d7dfc7d62854e39b458ed75007b33758f61d4